{{ cookiecutter.hosting_github_repo_name }}/SETUP/README.md

```diff
@@ -104,7 +104,7 @@ There are additional tasks required after generating this repository.
     This step should be run while within the `SETUP` directory (`cd SETUP`)
 
     ```bash
-    just setup {{ cookiecutter.firestore_region }}
+    just setup {{ cookiecutter.infrastructure_slug }} --region={{ cookiecutter.firestore_region }}
```

Comment:
I think `--region=` should be removed? For all the niceties of `just`, it doesn't have "named parameters" just positional :/